catroid/res/values/styles.xml

```diff
@@ -376,11 +397,6 @@
         <item name="android:background" >@drawable/brick_cyan_3h</item >
     </style >
 
-    <style name="BrickContainer.Phiro.Light" >
```

Comment:
@AdrianSch why did you remove the phrio styles? are they now contained within arduino styles?